portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java

```diff
@@ -0,0 +1,41 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.gson.JsonObject;
+
+/** Servlet that handles deleting data */
+@WebServlet("/delete-data")
+public class DeleteDataServlet extends HttpServlet {
+@Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String password = request.getParameter("password");
+    JsonObject jsonResponse = new JsonObject();
+
+    if (!password.equals("shuli-super-secret-password")) {
+      jsonResponse.addProperty("success", false);
+      jsonResponse.addProperty("errorMessage", "Incorrect password.");
+    }
+    else {
+      jsonResponse.addProperty("success", true);
```

Comment:
The success is indicated through HTTP status code, so this property will no longer be required.